tuf/signed/verify.go

```diff
@@ -77,10 +77,11 @@ func VerifySignatures(s *data.Signed, roleData data.BaseRole) error {
 		if key.ID() != sig.KeyID {
 			return ErrInvalidKeyID{}
 		}
-		if err := VerifySignature(msg, sig, key); err != nil {
+		if err := VerifySignature(msg, &sig, key); err != nil {
 			logrus.Debugf("continuing b/c %s", err.Error())
 			continue
 		}
+		s.Signatures[i].IsValid = sig.IsValid
```

Comment:
Using this method suggested by @cyli in the last comment if that's fine with all? @riyazdf @endophage